diff --git a/index.d.ts b/index.d.ts
index 2196aea21a679fe275cb67362167d586fd20ac0a..5a0e4ede7d2cd26c4b08c148c2064d074c9944b9 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -619,23 +619,23 @@ declare namespace SteamCommunity {
     type CallbackError = Error & { [key: string]: any } | null;
 
     /** Unique and can change after a trade. */
-    type assetid = number | string;
+    type assetid = string;
 
     type userid = SteamID | string;
 
     type appid = number;
 
     /** 2 for csgo... */
-    type contextid = number;
+    type contextid = string;
 
     /**
      * In a nutshell, a classid "owns" an instanceid. The classid is all you need to get a general overview of an item.
      * For example, items with the same classid will pretty much always have the same name and image.
      */
-    type classid = number;
+    type classid = string;
 
     /** An ID that describes an item instance that inherits properties from a class with the class id being noted in the instance (totally not unique). */
-    type instanceid = number;
+    type instanceid = string;
 
     type packageid = number | string;
 
