diff --git a/index.d.ts b/index.d.ts
index 9434e88a17a7a9315fa030330ba959eb5743f126..aac4d17597245bd632dd53cea8fd11384d3ca2d7 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -21,7 +21,7 @@ declare class TradeOfferManager extends EventEmitter {
     cancelOfferCount: number | null;
     cancelOfferCountMinAge: number;
     pollData: any;
-    readonly apiKey: string | null;
+    apiKey: string | null;
     steamID: SteamID | null;
     storage: FileManager;
 
@@ -689,6 +689,8 @@ declare namespace TradeOfferManager {
          * @param received An array of TradeOffer objects for offers we received
          */
         offerList: (filter: number, sent: TradeOffer[], received: TradeOffer[]) => void;
+
+        debug: (msg: string) => void;
     }
 
     interface TradeOfferManagerOptions {
